libevm/JitVM.cpp

```diff
@@ -175,9 +174,27 @@ void create(evm_result* o_result, ExtVMFace& _env, evm_message const* _msg) noex
 	}
 	else
 	{
-		o_result->code = EVM_FAILURE;
-		o_result->output_data = nullptr;
-		o_result->output_size = 0;
+		o_result->code = EVM_REVERT;
```

Comment:
It could fail for some other reason, like out of gas or address collision. Will this code affect anything?